### ISSUE TITLE
0 and empty default values of parameters not displaying

### ISSUE THREAD

**Issue by bchoang** [sourcey/spectacle#103]
**Problem:**
When a parameter has '0' or an empty string ("") as its default value, the value is not rendered.

For example, a parameter such as these:
```
- name: parameter1
in: query
type: string
default: ""
required: false

- name: parameter2
in: query
type: integer
default: 0
required: false
```

Won't display any information about default values.

**Expected:**
It should still display 'default: 0' or 'default: "" ' in the generated HTML file.

**Comment by svisser** [comment]
This also happens for `default: false`.

**Comment by yyamano** [comment]
@bchoang @svisser Could you try https://github.com/yyamano/spectacle/commit/3f2b595dacb879274935741f7cbf697bd07572ed ? I think the change fixes the issue.

**Comment by auscaster** [comment]
Hi guys, I just merged Yuji's PR into master. Let me know if it works and
we can close this one off.

**Comment by yyamano** [comment]
@auscaster Sorry for making you confused. It's not merged yet. 
I just sent a pull request.  https://github.com/sourcey/spectacle/pull/109

**Comment by auscaster** [comment]
Thanks, merged and updated the package `0.9.13`

**Comment by svisser** [comment]
Thanks for this!
